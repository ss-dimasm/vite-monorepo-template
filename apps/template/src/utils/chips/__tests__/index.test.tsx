import { render } from '@testing-library/react'
import { DisplayChip, LinkChip } from '..'

describe('DisplayChip', () => {
  it('should match a snapshot', () => {
    expect(render(<DisplayChip />).asFragment()).toMatchSnapshot()
  })
})

describe('LinkChip', () => {
  it('should match a snapshot', () => {
    expect(render(<LinkChip />).asFragment()).toMatchSnapshot()
  })
})
