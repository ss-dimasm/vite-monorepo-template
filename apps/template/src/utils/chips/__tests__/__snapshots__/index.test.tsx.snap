// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DisplayChip > should match a snapshot 1`] = `
<DocumentFragment>
  <div
    class="mocked-styled-0"
  />
</DocumentFragment>
`;

exports[`LinkChip > should match a snapshot 1`] = `
<DocumentFragment>
  <div
    class="mocked-styled-1"
  />
</DocumentFragment>
`;
